vnext/ReactWindowsCore/DevServerHelper.h

```diff
@@ -15,45 +15,68 @@ class DevServerHelper {
  public:
   DevServerHelper() = default;
 
-  static std::string get_WebsocketProxyUrl(const std::string &debugHost) {
-    return string_format(WebsocketProxyUrlFormat, GetDeviceLocalHost(debugHost).c_str());
+  static std::string get_WebsocketProxyUrl(const std::string &sourceBundleHost, const uint16_t sourceBundlePort) {
+    return string_format(WebsocketProxyUrlFormat, GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str());
   }
 
-  static std::string get_LaunchDevToolsCommandUrl(const std::string &debugHost) {
-    return string_format(LaunchDevToolsCommandUrlFormat, GetDeviceLocalHost(debugHost).c_str());
+  static std::string get_LaunchDevToolsCommandUrl(
+      const std::string &sourceBundleHost,
+      const uint16_t sourceBundlePort) {
+    return string_format(
+        LaunchDevToolsCommandUrlFormat, GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str());
   }
 
   static std::string get_BundleUrl(
-      const std::string &debugHost,
+      const std::string &sourceBundleHost,
+      const uint16_t sourceBundlePort,
       const std::string &jsbundle,
       const std::string &platform,
       const std::string &dev,
       const std::string &hot) {
     return string_format(
         BundleUrlFormat,
-        GetDeviceLocalHost(debugHost).c_str(),
+        GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str(),
         jsbundle.c_str(),
         platform.c_str(),
         dev.c_str(),
         hot.c_str());
   }
 
-  static std::string get_OnChangeEndpointUrl(const std::string &debugHost) {
-    return string_format(OnChangeEndpointUrlFormat, GetDeviceLocalHost(debugHost).c_str());
+  static std::string get_OnChangeEndpointUrl(const std::string &sourceBundleHost, const uint16_t sourceBundlePort) {
+    return string_format(OnChangeEndpointUrlFormat, GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str());
+  }
+
+  static std::string get_PackagerConnectionUrl(const std::string &sourceBundleHost, const uint16_t sourceBundlePort) {
+    return string_format(PackagerConnectionUrlFormat, GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str());
+  }
+
+  static std::string get_PackagerOpenStackFrameUrl(
+      const std::string &sourceBundleHost,
+      const uint16_t sourceBundlePort) {
+    return string_format(PackagerConnectionUrlFormat, GetDeviceLocalHost(sourceBundleHost, sourceBundlePort).c_str());
```

Comment:
Should this be PackagerOpenStackFrameUrlFormat?

Reply:
good catch